docker_images/ci-docs/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM python:slim
 
 RUN apt-get update \
-    && apt-get install -y enchant-2 git make latexmk texlive-latex-base texlive-fonts-recommended texlive-fonts-extra texlive-latex-extra texlive-xetex xindy graphviz
+    && apt-get install -y curl enchant-2 git make latexmk texlive-latex-base texlive-fonts-recommended texlive-fonts-extra texlive-latex-extra texlive-xetex xindy graphviz
 
-COPY requirements.txt /tmp/
-
-RUN pip3 install --requirement /tmp/requirements.txt
+# Download the requirements from the conan docs repo so they stay in sync when we regenerate the image
+RUN curl -o /tmp/requirements.txt https://raw.githubusercontent.com/conan-io/docs/master/requirements.txt \
+    && pip3 install --requirement /tmp/requirements.txt
```

Comment:
```suggestion
# Install the requirements from the conan docs repo so they stay in sync when we regenerate the image
RUN pip3 install --requirement https://raw.githubusercontent.com/conan-io/docs/master/requirements.txt
```

You can run pip with an external url directly. As you would need to download, networking will be a requirement always.